superset-frontend/src/dashboard/components/dnd/DragDroppable.jsx

```diff
@@ -73,6 +74,64 @@ const defaultProps = {
   dragPreviewRef() {},
 };
 
+const DragDroppableStyles = styled.div`
+  ${({ theme }) => css`
+    position: relative;
+
+    &.dragdroppable--dragging {
+      opacity: 0.2;
```

Comment:
The theme contains opacity stops we should leverage wherever it makes sense. It includes 10% and 35% stops at the moment, so maybe we should try one of those here in a separate PR? There are probably random opacities all over we could audit for.

Reply:
Ok, I'll see which fits better 👍